samples/General/enforce-tag-and-value/azurepolicy.parameters.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This should not be empty. It should have the parameter definitions, just like you have defined in your azurepolicy.policy.json:

```
{
  "tagName": {
    "type": "String",
    "metadata": {
      "displayName": "Tag Name",
      "description": "Name of the tag, such as 'environment'"
    }
  },
  "tagValue": {
    "type": "String",
    "metadata": {
      "displayName": "Tag Value",
      "description": "Value of the tag, such as 'production'"
    }
  }
}
```